src/Pos/CLI.hs

```diff
@@ -283,17 +265,40 @@ walletPortOption portNum help =
         <> Opt.value portNum
         <> Opt.showDefault
 
-ipPortOption :: NetworkAddress -> Opt.Parser NetworkAddress
-ipPortOption na =
+externalNetworkAddressOption :: Maybe NetworkAddress -> Opt.Parser NetworkAddress
+externalNetworkAddressOption na =
```

Comment:
This one definitely should go earlier to master than CSL-870, because 870 depends on this one.